src/server/api/resolvers/userResolver.ts

```diff
@@ -50,17 +49,17 @@ const resolvers = {
             return user;
         },
         changePassword: async (obj: any,
-                               args: { id: number, password: string },
-                               context: ResolverContext) => {
+                               args: { id?: number, password?: string },
+                               context: any) => {
```

Comment:
why did we get rid of the resolver context type?

Reply:
Not sure, the interface got removed in some merge lol but again I think apollo typing is something that we should look into